Update checklist.md (#126)
mate-academy/react_autocomplete: checklist.md

> - make sure that filter won't be called if user enred spaces only;

Reviewer: ```suggestion
- make sure that the filter is not called if user enters only spaces;
```
